etc/config-template.toml

```diff
@@ -308,7 +308,7 @@
 
 ## Represents the maximum number of threads that will concurrently perform a sub-compaction job by
 ## breaking it into multiple, smaller ones running simultaneously.
-# max-sub-compactions = 1
+# max-sub-compactions = 3
```

Comment:
seems the template is already out of sync with the actual max_background_jobs default.